tests/bluetooth/tester/src/btp/btp_has.h

```diff
@@ -0,0 +1,61 @@
+/* btp_has.h - Bluetooth tester headers */
+
+/*
+ * Copyright (c) 2023 Oticon
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* HAS commands */
+#define BTP_HAS_READ_SUPPORTED_COMMANDS	0x01
```

Comment:
Suggest to indent the values of the commands to be at the same column

Reply:
Done.